### PR TITLE
Update AmazonCorrettoMigrations.scala

### PR DESCRIPTION
**Pull request by dr29bart**
Updated jdk contains fix for https://github.com/corretto/corretto-11/issues/73

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/java/AmazonCorrettoMigrations.scala [hunk 1]
```diff
@@ -75,4 +75,32 @@ class AmazonCorrettoMigrations {
...
+        MacOSX,
+        Some(Amazon)
+      ),
+      Version(
+        "java",
+        "11.0.6-amzn",
+        "https://corretto.aws/downloads/resources/11.0.6.10.1/amazon-corretto-11.0.6.10.1-1-windows-x64-jdk.zip",
+        Windows,
+        Some(Amazon)
+      )
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/java/AmazonCorrettoMigrations.scala]: Is there really no update for Linux??

Comment (dr29bart) [reply src/main/scala/io/sdkman/changelogs/java/AmazonCorrettoMigrations.scala]: nope. Linux is OK: 11.0.6.10.1
those are hot-fixes for 
Windows -> 11.0.6.10.1-1.
MacOs -> 11.0.6.10.1-2.

Comment (marc0der) [reply src/main/scala/io/sdkman/changelogs/java/AmazonCorrettoMigrations.scala]: Cool, thanks for the clarification.

**Comment by marc0der** [comment]
Thanks for contributing!

**Pull request merged**